src/features/commands/ban.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
         ),
     async execute (client, interaction) {
         const userSnowflake = interaction.options.getUser('user');
-        if (interaction.member.permissions.serialize().BAN_MEMBERS) {
+        if (interaction.member.permissions.serialize().BAN_MEMBERS && interaction.member.roles.highest.position > interaction.options.getMember('user').roles.highest.position) {
```

Comment:
This doesn't seem to work if I try to ban the user which is outside of the server
![image](https://user-images.githubusercontent.com/11031051/133849188-5891d8c0-1005-4889-b2c1-cbf75c62ffe6.png)

Reply:
I will rewrite the whole ban command, because well you know, the code is messy and it's missing so many things